en.lproj/InfoPlist.strings

```diff
@@ -2,7 +2,9 @@
 
 NSHumanReadableCopyright = "Copyleft 2017, RIME Developers";
 
-im.rime.inputmethod.Squirrel = "Squirrel";
+im.rime.inputmethod.Squirrel.Rime = "Squirrel";
```

Comment:
Let's simplify `im.rime.inputmethod.Squirrel.Rime` to `im.rime.inputmethod.Squirrel`, drop `.Rime` everywhere in the IDs. because `im.rime.inputmethod.Squirrel` is also the bundle identifier, so this string can also serve as localized app name.

Reply:
Please revert this commit.

Reply:
I can take this in merging code.